### DISCUSSION TITLE
这是账号被标记了还是最近严打

### DISCUSSION THREAD

**Discussion by Miuzarte** [Mrs4s/go-cqhttp#1350] [General]
登上没够两分钟就又被冻，腾子未免
![Screenshot_20220202-190050_MT管理器.png](https://user-images.githubusercontent.com/66856838/152141682-05134196-fdfc-4cc7-af97-079ec9906920.png)

**Comment by Zhongren233** [comment]
看样子用了什么搜图bot
我觉得要不先把搜图插件关掉挂两天？

**Comment by soberkingomi** [comment]
好几天前我就这样了，一直死

**Comment by GLGDLY** [comment]
感觉是最近严打，我开了几个月也没事，忽然风控，还好我关得快不然可能也是冻结……
